src/app/login/login-form.component.ts

```diff
@@ -41,8 +42,7 @@ export class LoginFormComponent {
     };
     this.couchService.post('notifications', data)
       .subscribe((res) => {
-        console.log(res);
-      }, (error) => this.message = 'Error');
+      }, (error) => this.planetMessageService.showMessage('Error'));

```

Comment:
Let's not pass an error message from this to the message service.  If there is an error, it will replace the user created message and confuse the person into thinking something went wrong creating the user.